src/providers/wms/qgswmsprovider.cpp

```diff
@@ -743,20 +743,24 @@ QImage *QgsWmsProvider::draw( QgsRectangle const &viewExtent, int pixelWidth, in
       return image;
     }
 
-    QgsDebugMsgLevel( QStringLiteral( "layer extent: %1,%2 %3x%4" )
-                      .arg( qgsDoubleToString( mLayerExtent.xMinimum() ),
-                            qgsDoubleToString( mLayerExtent.yMinimum() ) )
-                      .arg( mLayerExtent.width() )
-                      .arg( mLayerExtent.height() ), 3
-                    );
+    QgsDebugMsg( QStringLiteral( "layer extent: %1,%2,%3,%4 %5x%6" )
```

Comment:
I also didn't review fully :) Can you put these back to QgsDebugMsgLevel? it's way to noisy to output here.

Reply:
Sure but please merge the backports and I'll make a followup to be backported after

Reply:
https://github.com/qgis/QGIS/pull/36422